src/pages/lib-v1/q/platform/[platform].mdx

```diff
@@ -36,34 +36,34 @@ import flutter2 from '/src/fragments/lib-v1/flutter.mdx';
 
 ## Amplify JavaScript
 
+<Callout warning>
+
+Welcome to the developer preview of **Amplify JavaScript v6**. This preview introduces functional interfaces, typescript improvements and improved NextJS SSR support  for Authentication, Analytics, and Storage.
+
+The team is working on the other categories for the v6 release. This developer preview is offered for you to try it out and provide feedback on the coming changes. 
+
+V6_TODO: Retool this callout and link to RFC
```

Comment:
I'm wondering if we should we comment this line

Reply:
Leaving it in so that it gets further attention before we merge it.